content/en/docs/getting-started/setup/create-cluster/google.md

```diff
@@ -7,8 +7,261 @@ publishdate: 2017-02-01
 lastmod: 2017-02-01
 weight: 50
 ---
-# Creating a Google cluster
 
+Google Cloud Platform (GCP) offers a powerful service for creating
+robust, highly-available clusters using the Kubernetes automated
+deployment platform. Google Kubernetes Engine (GKE) is a
+highly-available and secure kubernetes implementation that leverages the
+scalability of Google Cloud to provide a comprehensive solution when
+combined with the cloud-native development power and stability of
+Jenkins X.
+
+Cluster installation and configuration in Jenkins X is possible through
+*Jenkins X Boot*, a command that allows users to create Kubernetes
+clusters without running through various commands for configuring
+storage, pipeline, GitOps, and other resources.
+
+# Cluster prerequisites
+
+In order to create a cluster in Jenkins X, you must have the following
+configured:
+
+-   A Google Cloud Platform (GCP) account for creating kubernetes
+    clusters
+
+-   The Kubernetes command-line tool, which can be installed to your
+    local installation using the `jx install` command:
+
+        jx install dependencies -d kubectl
+
+-   Access to the [GitHub](https://www.github.com) remote Git provider,
+    which allows users to create organizations and multiple user
+    accounts:
+
+    -   An *Organization* from which you invite existing users (in this
+        example, the organization is called `acmecorp`). Create an
+        organization by clicking the **+** button at the top right
+        of the GitHub project page or through the [Create an
+        organization](https://github.com/organizations/new) page.
+
+    -   A GitHub user account for creating and managing development
+        repositories (for example `acmeuser`). Invite the user to your
+        organization.
+
+    -   A *bot* user that automates pull request notifications and
+        creates preview environments for quick validation and acceptance
+        for code merging. The bot account must have a token created by
+        your organization that authenticates the bot and allows it to
+        perform various tasks on the repositories within your
+        organization. In the examples, a GitHub account named `acmebot`
+        is used.
+
+# Creating a Kubernetes cluster
+
+The cluster creation process on Jenkins X(run via the command `jx boot`)
+requires performing a pre-configuration procedure to successfully
+complete the cluster installation process:
+
+1.  Run `jx create cluster` at the command-line to initialize the
```

Comment:
Why do we recommend the `jx create cluster` command? What's the benefit compared to `gcloud` or `terraform`? Shouldn't we guide people towards doing things the right way?